src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-fri calendar--month-29">
```

Comment:
[CHECKLIST]  2.[BEM] - Make sure to follow BEM naming convention for complex modifiers: 
block-name--modifier-name--modifier-value

Please add two dashes between modifier name and modifier value (calendar--start--fri)